src/clr-angular/layout/tabs/tabs.ts

```diff
@@ -88,12 +108,60 @@ export class ClrTabs implements AfterContentInit {
   }
 
   ngAfterContentInit() {
-    if (typeof this.ifActiveService.current === 'undefined') {
-      this.tabLinkDirectives.first.activate();
+    this.subscriptions.push(this.listenForTabLinkChanges(), this.listenForOverflowMenuFocusChanges());
+
+    if (typeof this.ifActiveService.current === 'undefined' && this.tabLinkDirectives[0]) {
+      this.tabLinkDirectives[0].activate();
     }
   }
 
   toggleOverflow(event: any) {
     this.ifOpenService.toggleWithEvent(event);
   }
+
+  checkFocusVisible() {
+    if (!this.ifOpenService.open && this.inOverflow()) {
+      this.ifOpenService.open = true;
+    } else if (this.ifOpenService.open && !this.inOverflow()) {
+      this.ifOpenService.open = false;
+    }
+  }
+
+  inOverflow() {
+    return (
+      this.tabLinkElements.indexOf(document.activeElement as HTMLElement) > -1 &&
```

Comment:
This could be simplified `this.tabLinkElements.includes(document.activeElement)`

Reply:
It is part of the original back-ported code. Actually, it can be considered as a note to me :) 
I remember I did it like this, because of questionable support of array.includes in the IE browser/s, though we later agreed that this gets handled gently at compilation time.